api/turing/validation/validator.go

```diff
@@ -287,6 +287,19 @@ func validateConditionOrthogonality(
 	}
 }
 
+func ValidateStdEnsemblerNotConfiguredForNopExpEngine(
```

Comment:
Can we flip the check and the error message around to make sure that if the experiment engine is nop, standard ensembler cannot be selected? This way, it will match the typical router configuration workflow more closely (i.e., fixing the experiment engine as it is chosen first).

Reply:
Also, this function can be private (name can start with lower case letter) as there is no need for it to be called explicitly from another package. Even the tests are using `validate.Struct` to invoke this function.

Reply:
Okay, I've flipped the error and the check around now, i.e.:
```go
	if experimentEngine.Type == models.ExperimentEngineTypeNop {
		if ensembler != nil && ensembler.Type == models.EnsemblerStandardType {
			sl.ReportError(ensembler.Type, "Type", "Type",
				"should not be standard if an nop experiment engine is configured", "")
		}
	}
```
Thanks for catching the part on the function being public too!